lib/osf-components/addon/components/delete-button/template.hbs

```diff
@@ -10,6 +10,18 @@
     >
         {{fa-icon 'times' size='lg'}}
     </Button>
+{{else if this.smallSecondary}}
+    <Button
+        data-analytics-name='Delete button'
+        data-test-delete-button
```

Comment:
Would it be worth it to have a more descriptive test selector, just in case there's another delete button on a page with these controls one day?